main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/net/html"
+	"golang.org/x/oauth2"
+)
+
+var (
+	ctx = context.Background()
```

Comment:
I think we should avoid this. For now, let's pass `context.Background()` to all things that need it, explicitly.

If the context we use changes in the future, it'll be easier to spot at a glance and adjust.